Source/scrollrt.cpp

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file plrmsg.cpp
+ *
+ * Implementation of functionality for rendering the dungions, monsters and calling other render routines.

```

Comment:
```suggestion
 * Implementation of functionality for rendering the dungeons, monsters and calling other render routines.
```

Same change for header file.